### PR TITLE
Fix: Update README to correct function signatures in examples

### PR DESCRIPTION
**Pull request by aokj4ck**
# Description

1. Update README example for SnapshotTest subclass 
    - Change to override `class` functions
2. Update README example for AccessibilityPreviewTest subclass
    - Remove underscore for `issue` parameter

Tested in Xcode 16.0.

### CODE REVIEW

File: README.md
Comment (aokj4ck) [inline README.md:51]: In this README the existing sample uses `SnapshotTest` and that works in my own separate project to perform the Preview layout test. 

Should DemoAppPreviewTest.swift file be changed to inherit from `SnapshotTest` instead of `PreviewLayoutTest`?

Comment (noahsmartin) [reply README.md:51]: There are two tests in that example, one that uses SnapshotTest and one that uses PreviewLayoutTest

https://github.com/EmergeTools/SnapshotPreviews/blob/main/Examples/DemoApp/DemoAppTests/DemoAppSnapshotTest.swift
https://github.com/EmergeTools/SnapshotPreviews/blob/main/Examples/DemoApp/DemoAppTests/DemoAppPreviewTest.swift

You can use either, the snapshot test one just creates an image (a snapshot) and the layout one just performs a layout pass but doesn't save a png

**Review by noahsmartin — APPROVED** [review]
Thanks @aokj4ck

File: README.md
Comment (aokj4ck) [reply README.md:51]: Brilliant and makes sense, thank you. From here I see "2 workflows awaiting approval. This workflow requires approval from a maintainer." and a blocking "Only those with write access to this repository can merge pull requests" so please merge whenever you like.

**Pull request merged**
